ilastik/applets/labeling/labelingGui.py

```diff
@@ -871,3 +890,13 @@ def allowDeleteLastLabelOnly(self, enabled):
         assume that all previous cell counts are given.
         """
         self._allowDeleteLastLabelOnly = enabled
+
+    def defTwoInitialLabels(self, enabled):
+        # in some workflows wen need always at least two labels to function correctly.
+        # setting enabled to True restricts having less than two labels
+        self._addNewLabel()
+        self._addNewLabel()
+        self.labelingDrawerUi.labelListModel.makeRowPermanent(0)
+        self.labelingDrawerUi.labelListModel.makeRowPermanent(1)
+
+        self._defTwoInitialLabels = enabled
```

Comment:
if above comment is true, this could have a more catchy name, like: `forceAtLeastTwoLabels`